deployments/engine/templates/scaledobject.yaml

```diff
@@ -0,0 +1,26 @@
+apiVersion: keda.sh/v1alpha1
```

Comment:
Can you add a helm variable here and create this resource when enabled? The argument is same as `ServiceMonitor`. We expect Keda should be installed, but it's nice to have an option to disable it.